env_collaction.template

```diff
@@ -3,10 +3,10 @@ DbHost=db
 DbUser=postgres
 DbPort=5432
 Db=CollActionDb
-FromName=
 FromAddress=your@email
-SendGridUser=
-SendGridKey=
+SesAwsAccessKey=
+SesAwsAccessKeyId=
+SesRegion=
```

Comment:
Maybe update the 'Getting CollAction running locally' with the Ses configurations.